base/set.jl

```diff
@@ -147,9 +162,20 @@ function unique(f::Callable, C)
 end
 
 """
-    allunique(itr)
+    allunique(itr) -> Bool
+
+Return `true` if all values from `itr` are distinct when compared with [`isequal`](@ref).
```

Comment:
Reminding me that I liked the `alldistinct` naming better, if we're even going to describe it that way in the docstring